android/src/main/java/com/balthazargronon/RCTZeroconf/ZeroconfModule.java

```diff
@@ -4,7 +4,9 @@
 import android.net.nsd.NsdManager;
 import android.net.nsd.NsdServiceInfo;
 import android.net.wifi.WifiManager;
+import android.os.Build;
```

Comment:
Small question, why this import was required?

Reply:
This is a good question. Maybe @makeitnew could answer?

I have a feeling it can be removed.

Reply:
Having looked through the code, I don't think it's necessary. Unfortunately, I'm not able to verify this. I'll remove it anyway as the code does seem to not use the import.